### PR TITLE
fix validation errors and warnings

### PR DESCRIPTION
**Pull request by Bolaji06**
Fixed all the errors and warnings. Available for review @Mavludin

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,19 +24,18 @@
...
 
-    <link rel="stylesheet" href="assets/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="assets/css/bootstrap.min.css">
 
-    <link rel="stylesheet" href="assets/css/style.css" />
+    <link rel="stylesheet" href="assets/css/style.css">
   </head>
 
-  <body data-spy="scroll" data-target=".navbar" data-offset="40" id="home">
+  <body data-spy="scroll" data-target=".navbar" data-offset="40" id="home-body">
```
Comment (Mavludin) [inline index.html:34]: @Bolaji06 Did you check that nothing breaks on UI with this ID change?

File: index.html [hunk 2]
```diff
@@ -748,7 +733,7 @@ <h6>Контакты</h6>
       <div class="modal-dialog modal-dialog-scrollable">
         <div class="modal-content">
           <div class="modal-header">
-            <h5 class="modal-title" id="exampleModalLabel">Путь Frontend разработчика</h5>
+            <h5 class="modal-title" id="reactExampleModalLabel">Путь Frontend разработчика</h5>
```
Comment (Mavludin) [inline index.html:736]: We must be sure that whether we need to do some changes in `style.css` as well or not

File: index.html (hunk 1 above)
Comment (Bolaji06) [reply index.html:34]: Did a good checks on the UI and made sure every part of the UI works as it was before.

File: index.html (hunk 2 above)
Comment (Bolaji06) [reply index.html:736]: Check the style.css whether their is a usage of it there, but I can't find none.

**Review by Mavludin — APPROVED** [review]
LGTM. Thanks for the help @Bolaji06

**Pull request merged**